xml/depl_nodes.xml

```diff
@@ -2306,7 +2297,7 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>User Name / Password for accessing the ECOM server</guimenu>
+    <term><guimenu>User Name</guimenu> / <guimenu>Password for accessing the ECOM server</guimenu>
```

Comment:
In the Web UI, the two fields are named: 'Username for accessing the ECOM server' and 'Password for accessing the ECOM server'.